common/djangoapps/contentserver/middleware.py

```diff
@@ -146,6 +148,13 @@ def process_request(self, request):
             response['Content-Type'] = content.content_type
             response['Last-Modified'] = last_modified_at_str
 
+            # Set CORS headers here.  Some courses pull assets via XHR or load them in iframes,
+            # so we want to definitively say that this is allowed.
+            base_url = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+            response['Access-Control-Allow-Origin'] = base_url
+            response['Content-Security-Policy'] = "frame-ancestors 'self' {base_url}".format(base_url=base_url)
+            response['X-Frame-Options'] = "ALLOW-FROM {base_url}".format(base_url=base_url)
```

Comment:
https://www.veracode.com/blog/2014/03/guidelines-for-setting-security-headers implies it needs the scheme.  

settings.SITE_NAME doesn't contain the scheme, so we'll need to add it.